doc/development_guide.md

```diff
@@ -113,6 +113,7 @@ General development steps that apply to both a virtual environment or a global e
 
 cd <workspaceRoot>
 python -m pip install --upgrade pip
+python -m pip install -r requirements-dev.txt

```

Comment:
this should be `python dev_setup.py`, which runs the above command in addition to some other operations.

Reply:
Applies to the other lines as well.

Reply:
Yes, turns out this should run in our build.py so calling this again shouldn't be necessary. I'll close this PR and not apply the changes.

Reply:
No, this does need to be added, otherwise dependencies needed to run build.py won't be installed.

Reply:
@pensivebrian what about line 55 in the build.py file? It seemed that the goal of this file was to install dependencies as well (although it hasn't worked for me).

Reply:
That's for building platform specific wheels when building rpm and deb package for Linux. There is a hook to invoke a custom version of PIP. We probably could document this a lot better. However, I am wondering if that should be requirements.txt (we don't have this but should) and not dev-requirements.txt. Could be a bug there.

Reply:
Re-opened and made new commit with suggested changes. I'm still running into Mac-related development issues, but this appears to be unrelated.